apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: limits
+title: Realtime Limits
+description: Realtime Rate Limiting
+---
+
+Since deploying a global multi-tenant cluster for Supabase Realtime, we've implimented some rate limits to protect our infrastructure.
+
+Rate limits are mostly configurable per project and our cluster can support millions of concurrent connections if needed. If you're running into any rate limiting and it's causing issues, please [contact support](https://app.supabase.com/support/new).
+
+## Limits by Plan
+
+Limits are set on the Free and Pro plans accordingly. See our [pricing page](https://supabase.com/pricing) for more information.
+
+Above the Pro plan Realtime is billed on a usage basis. We still employ limiting above the Pro plan to protect our infrastructure. If you're on an Enterprise plan simply contact support and we'll increase your limits as needed.
+
+Enterprise plan limits start at:
+
+- 500 concurrent users
+- 1,000 messages per second
+- 500 concurrent Channels
```

Comment:
Is this per client? If so, is it not 100?

https://github.com/supabase/multiplayer/blob/c443f5af0c7ae9dca8f646720fa2e2f074916e91/lib/realtime_web/channels/realtime_channel.ex#L51

Reply:
Yeah that's 100 per client.  We're not currently limiting Channels per tenant but we want to eventually. 

I could add that for clarity.

Reply:
Oh ya I added it below.